docs/installation/sources/sources_linux.rst

```diff
@@ -322,6 +323,215 @@ There are two possibilities:
+    `CMake specific arguments <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_
+    page of the colcon_ manual.
+
+Run an application
+^^^^^^^^^^^^^^^^^^
+
+When running an instance of an application using *Fast DDS Python bindings*, the colcon overlay built in the
+dedicated :code:`Fast-DDS-python` directory must be sourced.
+There are two possibilities:
+
+* Every time a new shell is opened, prepare the environment locally by typing the
+  command:
+
+  .. code-block:: bash
+
+      source ~/Fast-DDS-python/install/setup.bash
+
+* Add the sourcing of the colcon overlay permanently to the :code:`PATH`, by typing the following:
+
+  .. code-block:: bash
+
+      echo 'source ~/Fast-DDS-python/install/setup.bash' >> ~/.bashrc
+
+.. _cmake_installation_python_linux:
+
+CMake installation
+------------------
+
+This section explains how to compile *Fast DDS Python bindings* with CMake_, either
+:ref:`locally <local_installation_python_sl>` or :ref:`globally <global_installation_python_sl>`.
+
+.. _local_installation_python_sl:
+
+Local installation
+^^^^^^^^^^^^^^^^^^
+
+#. Create a :code:`Fast-DDS-python` directory where to download and build *Fast DDS Python bindings* and its
+   dependencies:
+
+   .. code-block:: bash
+
+       mkdir ~/Fast-DDS-python
+
+#. Clone the following dependencies and compile them using CMake_.
+
+   * `Foonathan memory <https://github.com/foonathan/memory>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS-python
+         git clone https://github.com/eProsima/foonathan_memory_vendor.git
+         mkdir foonathan_memory_vendor/build
+         cd foonathan_memory_vendor/build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-python/install -DBUILD_SHARED_LIBS=ON
+         sudo cmake --build . --target install
+
+   * `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS-python
+         git clone https://github.com/eProsima/Fast-CDR.git
+         mkdir Fast-CDR/build
+         cd Fast-CDR/build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-python/install
+         sudo cmake --build . --target install
+
+   * `Fast DDS <https://github.com/eProsima/Fast-DDS.git>`_
+
+     .. code-block:: bash
+
+         cd ~/Fast-DDS-python
+         git clone https://github.com/eProsima/Fast-DDS.git
+         mkdir Fast-DDS/build
+         cd Fast-DDS/build
+         cmake .. -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-python/install -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-python/install
+         sudo cmake --build . --target install
+
+#. Once all dependencies are installed, install *Fast DDS Python bindings*:
+
+   .. code-block:: bash
+
+       cd ~/Fast-DDS-python
+       git clone https://github.com/eProsima/Fast-DDS-python.git
+       mkdir Fast-DDS-python/build
+       cd Fast-DDS-python/build
+       cmake ..  -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-python/install -DCMAKE_PREFIX_PATH=~/Fast-DDS-python/install
```

Comment:
This command fails since the CMakeLists.txt files are a bit deeper in the tree:

`CMake Error: The source directory "/home/user/Fast-DDS-python/Fast-DDS-python" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
`

Reply:
Done